### PR TITLE
steps: new version with batch version of tets functions

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Comment by pramodk** [comment]
You have to also change steps version in: deploy/packages/bbp-packages.yaml

File: var/spack/repos/builtin/packages/steps/package.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Steps(CMakePackage):
     git      = "git@github.com:CNS-OIST/HBP_STEPS.git"
 
     version("develop", branch="master", submodules=True)
-    version("3.5.0a",  commit="30eb779", submodules=True)
+    version("3.5.0b",  commit="800ff45", submodules=True)
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/steps/package.py]: Tag with `-beta`?

**Review by pramodk — COMMENTED** [review]
I think Matthias meant create tag in repo instead of commit.

But as repo is not owned by BBP, it’s ok to have commit in this case.

**Comment by alkino** [comment]
So, I should remove -beta from my patch?

**Review by pramodk — APPROVED** [review]

**Comment by alkino** [comment]
In fact I will redo a version with fix from @smelchio

**Review by matz-e — APPROVED** [review]
Fine. But check that this will rebuild with the changed commit in the PR directory.

**Review by pramodk — APPROVED** [review]

**Pull request merged**